modules/phase_field/src/postprocessors/FeatureFloodCount.C

```diff
@@ -123,15 +126,17 @@ FeatureFloodCount::FeatureFloodCount(const InputParameters & parameters) :
     _maps_size(_single_map_mode ? 1 : _vars.size()),
     _n_procs(_app.n_processors()),
     _entities_visited(_vars.size()), // This map is always sized to the number of variables
+    _feature_counts_per_map(_maps_size),
     _feature_count(0),
+    _max_local_size(0),
```

Comment:
I can probably drop this variable altogether. It's now only being used for a vector.reserve() operation.